package-system/pyside2/build_linux.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+# 
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+# Get the python executable from the package dependency
+LOCAL_PYTHON37_BIN=$TEMP_FOLDER/python-3.10.5-rev1-linux/python/bin/python3
```

Comment:
We might want to change this variable to get rid of "37" from the name.

```suggestion
LOCAL_PYTHON3_BIN=$TEMP_FOLDER/python-3.10.5-rev1-linux/python/bin/python3
```

Reply:
Done